lib/slack-ruby-bot/commands/base.rb

```diff
@@ -82,6 +79,17 @@ def self.finalize_routes!
         return if self.routes && self.routes.any?
         command default_command_name
       end
+
+      def self.get_gif_and_send!(client, channel, text, keywords, options = {})
+        gif = begin
+          Giphy.random(keywords)
+        rescue StandardError => e
+          logger.warn "Giphy.random: #{e.message}"
+          nil
+        end
+        text = text + "\n" + gif.image_url.to_s if gif
```

Comment:
For a GIF this sends a blank line. Send `nil` for the GIF and then you can do `[text, gif].compact.join("\n")` instead of this whole thing.